cura/Backups/Backup.py

```diff
@@ -94,8 +97,7 @@ def _makeArchive(self, buffer: "io.BytesIO", root_path: str) -> Optional[ZipFile
         :param root_path: The root directory to archive recursively.
         :return: The archive as bytes.
         """
-
-        ignore_string = re.compile("|".join(self.IGNORED_FILES))
+        ignore_string = re.compile("|".join(self.IGNORED_FILES + self.IGNORED_FOLDERS))
```

Comment:
Note that ignore_string is later used to search/match through the entire absolute path. Technically that would mean that when someone installs Cura to C:/cache/ or D:/plugins/ the backup system won't work anymore.

This is already a risk w.r.t. the way it's done now, though except for cache those all contain dots or underscores. Nevertheless fixing this falls outside of the scope of this ticket, I think.